tests/geometry-calculator.test.js

```diff
@@ -0,0 +1,99 @@
+
+const { calculateDistance, calculateJunctionPoint, isPointOnLine } = require('../modules/geometry-calculator')
+const { Line, getPointByY, getPointByX } = require('../modules/ecs6-class/line');
+const { Point } = require('../modules/ecs6-class/point');
+const mockConstructorPoint = jest.fn(constructor);
+const mockConstructorLine = jest.fn(constructor);
+
```

Comment:
this  functions mocks only a constructor (it's okay to mock each time a different class in the same mock constructor)
build a real mock of ES6, and you shouldn't have to export all this prototypes from the modules